### PR TITLE
Updated create client page and updating form to use formik

### PR DESCRIPTION
**Pull request by paulmatchett**
The following changes have been carried out in this PR:

* Removing all update client functionality as this has been left in by mistake and not longer in use.
* Updating layout to a container / form / index structure.
* Updating the CreateClient form to use formik.

### CODE REVIEW

File: src/components/CreateClient/form.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+  mapPropsToValues: () => {
+    return ({
+      clientName: '',
+    });
+  },
+
+  // Custom sync validation
+  validate: ({ clientName }) => {
+    let errors = {};
+    if (clientName === '') {
```
Comment (d1wilko) [inline src/components/CreateClient/form.js:26]: is this the only validation we need?- nothing about numbers or max length etc?

File: src/components/CreateClient/form.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+  isValid: PT.bool.isRequired,
+  errors: PT.shape({
+    clientName: PT.string,
+  }),
+  touched: PT.shape({
+    clientName: PT.bool,
+  }),
+};
+
+export default FormikEnhancer(CreateClientForm);
```
Comment (d1wilko) [inline src/components/CreateClient/form.js]: need an empty line at the bottom here

**Review by d1wilko — CHANGES_REQUESTED** [review]
small questions :)

File: src/components/CreateClient/form.js (hunk 1 above)
Comment (paulmatchett) [reply src/components/CreateClient/form.js:26]: Going by what existed before implementing formik, this was the only validation required.

File: src/components/CreateClient/form.js (hunk 2 above)
Comment (paulmatchett) [reply src/components/CreateClient/form.js]: 👍

**Review by d1wilko — APPROVED** [review]

**Pull request merged**